tests/guest-tools/unix/test_guest_tools_unix.py

```diff
@@ -0,0 +1,124 @@
+import pytest
+from lib.common import wait_for, wait_for_not
+
+# Requirements:
+# - a two host XCP-ng pool >= 8.0 (the second host is only used for the migration test)
+# - a unix VM supported by the Linux/install.sh script of the guest tools ISO
```

Comment:
I like seeing this in the test file, maybe it should be a standard for new tests?

Reply:
Yes, probably. Note that in some tests we've used a more formal and probably better form, where we also specify what command line arguments or parameters in data.py you need to feed to the test.

Reply:
Yes we should normalize that as well.